akka-projection-jdbc/src/test/scala/akka/projection/jdbc/JdbcProjectionSpec.scala

```diff
@@ -73,21 +73,35 @@ object JdbcProjectionSpec {
 
   class PureJdbcSession extends JdbcSession {
 
+    val logger = LoggerFactory.getLogger(this.getClass)
```

Comment:
This logger can also stay, is test only, but can be useful in case of debugging with Travis logs.